hydra_server/server/network.py

```diff
@@ -344,6 +395,26 @@ def purge_network(ctx, network_id, purge_data):
         network.purge_network(network_id, purge_data, **ctx.in_header.__dict__)
         return 'OK'
 
+    @rpc(Integer, Unicode, _returns=Unicode)
+    def set_network_status(ctx, network_id, status):
+        """
+        Set status of network to the specified status character
+
+        Args:
+            network_id (int): The network to delete
+            status (char): A or X
+
+        Returns:
+            string: 'OK'
+
+        Raises:
+            ResourceNotFoundError: If the network is not found
+        """
+        #check_perm('edit_topology')
+        network.set_network_status(network_id, status.upper(), **ctx.in_header.__dict__)
```

Comment:
I've added a check in the function definition for this. I think we need to do another refactor later where we pass all flags through a function to produce a boolean, and update all relevant HB functions to accept booleans instead of chars. Bot not now :)